lib/note-toolbar/index.tsx

```diff
@@ -180,6 +191,7 @@ const mapStateToProps: S.MapState<StateProps> = ({
 
 const mapDispatchToProps: S.MapDispatch<DispatchProps> = {
   deleteNoteForever: actions.ui.deleteOpenNoteForever,
+  onNewNote: () => createNote(),
```

Comment:
fun fact! there's no need to create a new function closure here unless we specifically want to create a function which ignores its arguments (as can be more useful when passing it as above in a click callback, which otherwise would pass the click event as the first argument to `onNewNote`)

for this we can pass the function directly and the type system makes sure we don't accidentally pass a non-function value.

```ts
onNewNote: createNote,
```

Reply:
This is really helpful, thank you again!